docs/source/models.rst

```diff
@@ -159,7 +159,16 @@ class has four key functions:
 
 
 Models From Code
-^^^^^^^^^^^^^^^^
+----------------
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    model/models-from-code
+
+To **learn more about the Models From Code feature**, please visit `the deep dive guide <model/models-from-code.html>`_ for more in-depth explanation 
+and to see additional examples.
 
 .. note::
     The Models from Code feature is available in MLflow versions 2.12.2 and later. This feature is experimental and may change in future releases.
```

Comment:
(probably follow-up) shall we remove experimental mark for model-from-code, as we want to promote it more?

Reply:
follow-up. I think we can remove it prior to 2.16